tool/site/client-react/src/components/prescription-summary/prescriptionLevelDetails.tsx

```diff
@@ -0,0 +1,135 @@
+  getNumberOfRepeatsIssuedExtension, getPerformerSiteTypeExtension,
+  getRepeatInformationExtension
+} from "../../fhir/customExtensions"
+
+export function createPrescriptionLevelDetails(medicationRequest: MedicationRequest, communicationRequests?: Array<CommunicationRequest>): PrescriptionLevelDetailsProps {
+  const prescriptionId = medicationRequest.groupIdentifier.value
+
+  const repeatExtension = getRepeatInformationExtension(medicationRequest.extension)
+  const repeatsIssuedExtension = getNumberOfRepeatsIssuedExtension(repeatExtension.extension)
+  const repeatIssued = repeatsIssuedExtension.valueInteger
+  const repeatsAllowedExtension = getNumberOfRepeatsAllowedExtension(repeatExtension.extension)
+  const repeatAllowed = repeatsAllowedExtension.valueInteger
+
+  const authoredOn = formatDate()
+  const startDate = medicationRequest.dispenseRequest.validityPeriod?.start ?? new Date().toISOString().slice(0, 10)
+  const nominatedOds = medicationRequest.dispenseRequest?.performer?.identifier?.value || ""
+
+  const nominatedTypeExtension = getPerformerSiteTypeExtension(medicationRequest.dispenseRequest.extension)
+  const nominatedTypeCode = nominatedTypeExtension.valueCoding.code
+  const nominatedType = getPharmacyTypeText(nominatedTypeCode)
+
+  const patientInstruction = communicationRequests
+    .flatMap(communicationRequest => communicationRequest.payload)
+    .filter(Boolean)
+    .filter(isContentStringPayload)
+    .map(payload => payload.contentString)
+    .join("\n")
+
+  return {
+    prescriptionId,
+    repeatIssued,
+    repeatAllowed,
+    authoredOn,
+    startDate,
+    nominatedOds,
+    nominatedType,
+    patientInstruction
+  }
+}
+
+export interface PrescriptionLevelDetailsProps {
+  prescriptionId: string
+  repeatIssued?: number
+  repeatAllowed?: number
+  authoredOn: string
+  startDate: string
+  nominatedOds?: string
+  nominatedType?: string
+  patientInstruction?: string
+}
+
+const PrescriptionLevelDetails = ({
+  prescriptionId,
+  repeatIssued,
+  repeatAllowed,
+  authoredOn,
+  startDate,
+  nominatedOds,
+  nominatedType,
+  patientInstruction
+}: PrescriptionLevelDetailsProps): JSX.Element => {
+
+  return (
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    <SummaryList>
+      <SummaryList.Row>
+        <SummaryList.Key>ID</SummaryList.Key>
+        <SummaryList.Value>{prescriptionId}</SummaryList.Value>
+      </SummaryList.Row>
+      {repeatAllowed > 1 &&
+        <>
+          <SummaryList.Row>
+            <SummaryList.Key>Repeat Information</SummaryList.Key>
+            <SummaryList.Value>{repeatIssued}/{repeatAllowed}</SummaryList.Value>
+          </SummaryList.Row>
+        </>
+      }
+      <SummaryList.Row>
+        <SummaryList.Key>Authored on</SummaryList.Key>
+        <SummaryList.Value>{authoredOn}</SummaryList.Value>
+      </SummaryList.Row>
+      <SummaryList.Row>
+        <SummaryList.Key>Effective date</SummaryList.Key>
```

Comment:
We're using title case everywhere else, so change to "Effective Date" for consistency.

Reply:
"Authored On" is up for debate though - https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case :shrug: